examples/Bickley_jet_shallow_water.jl

```diff
@@ -246,23 +245,24 @@ I = 6000:7000
 degree = 1
 linear_fit_polynomial = fit(t[I], log.(norm_v[I]), degree)
 
-best_fit = exp.(linear_fit_polynomial[0] .+ linear_fit_polynomial[1] * t)
-
-plt = plot(t, norm_v,
-            yaxis = :log,
-            ylims = (1e-3, 30),
-               lw = 4,
-            label = "norm(v)", 
-           xlabel = "time",
-           ylabel = "norm(v)",
-            title = "growth of perturbation norm",
-           legend = :bottomright)
-plot!(plt, t[I], 2 * best_fit[I], # factor 2 offsets our fit from the curve for better visualization
-               lw = 4,
-            label = "best fit")
+best_fit = @. exp(linear_fit_polynomial[0] + linear_fit_polynomial[1] * t)
+
+plot(t, norm_v,
+        yaxis = :log,
+        ylims = (1e-3, 30),
+           lw = 4,
+        label = "norm(v)", 
+       xlabel = "time",
+       ylabel = "norm(v)",
+        title = "growth of perturbation norm",
+       legend = :bottomright)
+
+plot!(plt, t[I], 2 * best_fit[I], # factor 2 offsets fit from curve for better visualization
+           lw = 4,
+        label = "best fit")
             
-# We can compute the slope of the curve on a log scale, which approximates the growth rate
-# of the simulation. This should be close to the theoretical prediction.
+# We can compute the slope of the curve on a logarithmic scale, which approximates the growth 
+# rate of the simulation. This should be close to the theoretical prediction.
 
-println("Growth rate in the simulation is approximated to be ", linear_fit_polynomial[1], ",\n",
+println("Numerical growth rate is approximated to be ", linear_fit_polynomial[1], ",\n",
         "which is close to the theoretical value of 0.14.")
```

Comment:
Some could say is equal to the theoretical value. Never found that before but here we do

Reply:
Shall I change it to `very close`?

Reply:
Sure